### DISCUSSION TITLE
link for latest deployment for branch containing a forward slash

### DISCUSSION THREAD

**Discussion by eps1lon** [vercel/community#970] [Help] [answered]
If I understood https://vercel.com/docs/concepts/deployments/generated-urls#url-with-git-branch correctly, for a branch named `beta-testing` on a fork the permalink would be https://beta-reactjs-git-fork-beta-testing-fbopensource.vercel.app.

How would that look like when the branch contains a foward slash e.g. `beta/testing` (example: https://github.com/reactjs/reactjs.org/pull/5193)? The following attempts don't work:
- https://beta-reactjs-git-fork-beta-testing-fbopensource.vercel.app
- https://beta-reactjs-git-fork-beta/testing-fbopensource.vercel.app
- https://beta-reactjs-git-fork-beta%2Ftesting-fbopensource.vercel.app

**Comment by amyegan** [comment, marked as answer]
Hi @eps1lon!  It looks like you're missing the `-org` part of the project name, and the `<branch-name>` portion of the URL is using the fork username from the preview deployment. So it would be http://beta-reactjs-org-git-fork-eps1lon-beta-testing-fbopensource.vercel.app/

Seems like something we need to make clear in the docs. Thanks for pointing this out!
